deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```diff
@@ -136,11 +136,9 @@ spec:
             {{- toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
-      {{- with .Values.cainjector.nodeSelector }}
+      {{- with (coalesce .Values.cainjector.nodeSelector .Values.global.nodeSelector) }}
       nodeSelector:
-        {{- range $key, $value := . }}
-        {{ $key }}: {{ $value | quote }}
-        {{- end }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
I don't think this needs to change as it will break the motivation for https://github.com/cert-manager/cert-manager/pull/7579.